hibernate-core/src/main/java/org/hibernate/loader/Loader.java

```diff
@@ -1610,7 +1607,7 @@ private void checkVersion(
 			else {
 				//If the object is already loaded, return the loaded one
 				object = session.getEntityUsingInterceptor( key );
-				if ( object != null ) {
+				if (object != null) {
```

Comment:
It is a pattern to get rid of space char uses previously advised. Is that a new format pattern to advocate then?

Reply:
No it is not, I'm going to fix it, thanks